packages/office-ui-fabric-react/src/components/List/List.types.ts

```diff
@@ -68,6 +68,12 @@ export interface IListProps extends React.HTMLAttributes<List | HTMLDivElement>
    */
   onRenderCell?: (item?: any, index?: number, isScrolling?: boolean) => React.ReactNode;
 
+  /**
+   * Optional callback for monitoriing when rendering has beed completed due to change in List pages.
+   * This can be on initial mount or on re-render due to scrolling.
+   */
+  onRenderComplete?: (pages: IPage[]) => void;
```

Comment:
my only request would be the name of this prop. onRenderComplete sounds like a 1 time deal. I'd suggest using onUpdatedPages or onPagesUpdated. This matches the method it is placed into.

Reply:
+@dzearing since he suggested current name. Maybe we could have both ? rename current behavior to `onPagesUpdated` and add have `onRenderComplete` to only fire once?

Reply:
Yeah, i'm a little confused (from naming) if onRenderComplete and onPagesUpdated were the same thing or not. Does one get called once, and one called several times?

Reply:
Yeah that would be the proposal:

`onRenderComplete` => called only once
`onPagesUpdated` => called multiple times (even 1st time as well)

Or would `onRenderComplete` be pointless at that point?